platforms/eth.py

```diff
@@ -25,4 +25,7 @@ def platform_config_update(config):
             (0, 3): 'veth7',
         }
 
+    cpu_port = config["cpu_port"]
```

Comment:
This is too specific and should not be part of the core code. Please use this feature to specify the CPU port veth: https://github.com/p4lang/ptf#passing-parameters-to-tests